src/App/pages/RepoPage/components/Card/Card.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Text from "../Text";
+import Star from './Star'
+import { Link } from 'react-router-dom';
+export type CardProps = {
+    className?: string,
+    image: string;
+    captionSlot?: React.ReactNode;
+    title: React.ReactNode;
+    subtitle?: React.ReactNode;
+    contentSlot?: React.ReactNode;
+    onClick?: React.MouseEventHandler;
+    actionSlot?: React.ReactNode;
+    dateSlot?: string
+    id: number
+};
+
+const Card: React.FC<CardProps> = ({ className, image, captionSlot, title, subtitle, contentSlot, onClick, actionSlot, dateSlot, id }) => {
+    return (
+        <Link to={`/repo/${id}`} className='repo-card__link'>
+            <div className={className + " card-icon"} onClick={onClick}>
```

Comment:
```suggestion
            <div className={cn(className, "card-icon")} onClick={onClick}>
```

import cn from 'clsx';